src/components/Tipseen/TipseenWizard.tsx

```diff
@@ -5,15 +5,18 @@ import Button from "../../components/Button/Button";
 import TipseenBasicContent from "./TipseenBasicContent";
 import styles from "./TipseenWizard.module.scss";
 
+const FINISH_TEXT = "Got it";
```

Comment:
This is a bit problematic, since if someone doesn't supply a value for it, it won't be translated even if all other strings are translated

Reply:
How does it behave today?

Reply:
Ok I see it also the behavior today. We should think about it, since it means we're not allowing devs to translate it.
Can it be in the `StepsConstants`, same as the `Next`, `Back`

Reply:
Yes I talked about it with Meirav, it is problematic but we are letting users override it with any string (using *ButtonProps)

Reply:
ok cool, np